lib/srv/discovery/common/watcher.go

```diff
@@ -180,6 +180,12 @@ func (w *Watcher) fetchAndSend() {
 					staticLabels[types.CloudLabel] = c
 				}
 
+				// Set the discovery type label to provide information about the
+				// matcher type that matched the resource.
+				if t := lFetcher.FetcherType(); t != "" {
+					staticLabels[types.DiscoveryTypeLabel] = t
```

Comment:
We should update the godoc for this const
https://github.com/gravitational/teleport/blob/1329276bbb9f57324d7540b11119803375c8b16d/api/types/constants.go#L747-L748